packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -12,20 +47,108 @@ export default {
     name: 'ImageTile',
     components: {},
     props: {
+        href: {
+            type: String,
+            default: null
+        },
+        tileId: {
+            type: String,
+            default: null
+        },
+        isSelected: {
+            type: Boolean,
+            default: false
+        },
+        isLink: {
+            type: Boolean,
+            default: false
+        },
+        displayText: {
+            type: String,
+            default: null
+        },
+        imgSrc: {
+            type: String,
+            default: null
+        }
+    },
+    data () {
+        return {
+            isFilterSelected: false
+        };
+    },
+    watch: {
+        isSelected (newValue) {
+            if (newValue !== this.isFilterSelected) {
+                this.isFilterSelected = newValue;
+            }
+        }
+    },
+    mounted () {
+        this.isFilterSelected = this.isSelected;
```

Comment:
This lifecycle method is client only, I believe. Therefore it'll cause issues during SSR on any consuming apps I believe. Might be worth looking into `created` or others. [Reference](https://ssr.vuejs.org/guide/universal.html#component-lifecycle-hooks)

Reply:
In fact, I think the `watch` method may take care of this for you - worth checking!

Reply:
I think:

```
watch: {
  isSelected: {
    immediate: true,
    handler (newValue) {
      if (newValue !== this.isFilterSelected) {
        this.isFilterSelected = newValue;
      }
    }
  }
}
```

Might work

Reply:
Cool, let me add this to the relevant ticket and I can explore that further there 👍